src/main/java/io/kestra/plugin/notifications/telegram/TelegramSend.java

```diff
@@ -0,0 +1,93 @@
+package io.kestra.plugin.notifications.telegram;
+
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.Task;
+import io.kestra.core.models.tasks.VoidOutput;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.notifications.telegram.api.TelegramBot;
+import io.kestra.plugin.notifications.telegram.api.TelegramBotImpl;
+import io.micronaut.http.client.DefaultHttpClientConfiguration;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.netty.DefaultHttpClient;
+import io.micronaut.http.client.netty.NettyHttpClientFactory;
+import io.micronaut.http.codec.MediaTypeCodecRegistry;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.slf4j.Logger;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Objects;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+        title = "Send input as a telegram message"
+)
+public class TelegramSend extends Task implements RunnableTask<VoidOutput> {
+    private static final String TELEGRAMAPI_BASE_URL = "https://api.telegram.org";
+    private static final NettyHttpClientFactory FACTORY = new NettyHttpClientFactory();
+    @Schema(
+            title = "Telegram Bot token",
+            requiredMode = Schema.RequiredMode.REQUIRED
```

Comment:
We use the `@NotNull` annotation for that, it will do the same on the JSON Schema